templates/taxi/car_detail.html

```diff
@@ -3,20 +3,31 @@
 {% block content %}
   <h1>
     {{ car.model }}
-    <a href="{% url 'taxi:car-delete' pk=car.id %}" class="btn btn-danger link-to-page">
-      Delete
-    </a>
-
-    <a href="{% url 'taxi:car-update' pk=car.id %}" class="btn btn-secondary link-to-page">
-      Update
-    </a>
   </h1>
   <p>Manufacturer: ({{ car.manufacturer.name }}, {{ car.manufacturer.country }})</p>
+  {% if user not in car.drivers.all %}
+    <form action = "{% url "taxi:driver-assign" car.id %}" method="post">
+      {% csrf_token %}
+      <button class = "btn btn-success link-to-page">Assign me to this car</button>
+    </form>
+  {% else %}
+    <form action = "{% url "taxi:driver-remove" car.id %}" method="post">
```

Comment:
Ensure that the `assign_driver` view correctly handles the POST request for removing the user from the car. The URL pattern and view logic should be verified to ensure they match the expected behavior.